### DISCUSSION TITLE
MODFLOW input files using ModelMuse?

### DISCUSSION THREAD

**Discussion by offgiancarlo** [pygsflow/pygsflow#40] [General]
PyGsflow scripts were developed with an interface for Flopy. 
However, is it possible to generate MODFLOW input files using ModelMuse to run pyGsflow?

**Comment by jlarsen-usgs** [comment]
Yes, MODFLOW Input files that were generated with ModelMuse can be loaded with both pyGSFLOW and FloPy. To load the modflow input files with pyGSFLOW:

```python
from gsflow.modflow import Modflow

model_ws = "path/to/my/model"
name_file = "model_name_file.nam"

ml = Modflow.load( name_file, model_ws=model_ws)
```
